src/functions-and-arrays.js

```diff
@@ -1,19 +1,73 @@
 // Iteration #1: Find the maximum
 
+const maxOfTwoNumbers = (number1, number2) => {
+  if (number1 > number2) {
+    return number1;
+  } else {
+    return number2;
+  }
+}
+
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
+function findLongestWord(arrayOfWords) {
+  let word = "";
+  for (let i = 0; i < arrayOfWords.length; i++) {
+    if (word.length < arrayOfWords[i].length) {
+      word = arrayOfWords[i];
+    }
+  }if (!arrayOfWords.length) {
+    return null
+  };
+    
+return word;
+}
+
 // Iteration #3: Calculate the sum
 
 const numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
+const sumNumbers = (arrayOfNumbers) => {
+let sum = 0; 
+for (let i = 0; i < arrayOfNumbers.length; i++) {
+  sum += arrayOfNumbers[i];
+}
+return sum;
+}
+
 // Iteration #4: Calculate the average
 // Level 1: Array of numbers
 const numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
 
+const averageNumbers = numbers => {
+  if (!numbers.length) {
+    return null;
+  }
+  return sumNumbers(numbers) / numbers.length;
+}
+
 // Level 2: Array of strings
 const wordsArr = ['seat', 'correspond', 'linen', 'motif', 'hole', 'smell', 'smart', 'chaos', 'fuel', 'palace'];
 
+const countWords = wordsArr => {
```

Comment:
Is this function counting the _words_ in the array, or the amount of _characters_?